src/app/app-routing.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { AppComponent } from './app.component';
+import { IndexComponent } from './components/index/index.component';
+import { ItinerarylistComponent } from './components/itinerary/itinerarylist/itinerarylist.component';
+import { NotFoundComponent } from './components/not-found/not-found.component';
+import { LoginComponent } from './components/operations/login/login.component';
+import { RegisterComponent } from './components/operations/register/register.component';
+
+const routes: Routes = [
+  //{path:'',component: AppComponent,pathMatch: 'full'},
+  {path: '' , component: IndexComponent},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
  {path: "" , component: IndexComponent},
```